.github/workflows/run_data_sync.yml

```diff
@@ -146,9 +169,26 @@ jobs:
           python scripts/gen_svg.py --from-db --year $(date +"%Y")  --language zh_CN --title "$(date +"%Y") Running" --type github --athlete "${{ env.ATHLETE }}" --special-distance 10 --special-distance2 20 --special-color yellow --special-color2 red --output assets/github_$(date +"%Y").svg --use-localtime --min-distance 0.5
 
       - name: Push new runs
+        if: env.SAVE_DATA_IN_GITHUB_CACHE != 'true'
         run: |
           git config --local user.email "${{ env.GITHUB_EMAIL }}"
           git config --local user.name "${{ env.GITHUB_NAME }}"
           git add .
           git commit -m 'update new runs' || echo "nothing to commit"
           git push || echo "nothing to push"
+
+      - name: Set Output
+        id: set_output
+        run: |
+          echo "SAVE_DATA_IN_GITHUB_CACHE=${{ env.SAVE_DATA_IN_GITHUB_CACHE }}" >> "$GITHUB_OUTPUT"
+          echo "DATA_CACHE_PREFIX=${{ env.DATA_CACHE_PREFIX }}" >> "$GITHUB_OUTPUT"
+
+  publish_github_pages:
```

Comment:
not all users use GitHub Page to deploy

Reply:
Actually, in the previous commits on my own branch, there was indeed a variable called `BUILD_GH_PAGES`, which controlled whether to build GitHub Pages. Considering that users building on Vercel with caching enabled need to understand this variable, I decided to add it back and provide an explanation in the documentation.

Reply:
commit: 64bbf77f7b134...64c63b2